apix-integrationtests-server/src/test/java/eu/xenit/apix/server/AlfrescoApplicationContextIT.java

```diff
@@ -0,0 +1,57 @@
+package eu.xenit.apix.server;
+
+import static org.junit.Assert.assertNotNull;
+
+import com.github.ruediste.remoteJUnit.client.RemoteTestRunner;
+import org.alfresco.service.ServiceRegistry;
+import org.alfresco.service.cmr.repository.NodeService;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.BeansException;
+import org.springframework.context.ApplicationContext;
+
+@RunWith(RemoteTestRunner.class)
+public class AlfrescoApplicationContextIT {
+    private static final Logger logger = LoggerFactory.getLogger(AlfrescoApplicationContextIT.class);
+    private ApplicationContext testApplicationContext;
+    private NodeService nodeService;
+    private ServiceRegistry serviceRegistry;
+
+    @Before
+    public void setUp() {
+        this.testApplicationContext = ApplicationContextProvider.getApplicationContext();
+
+        try{
+            serviceRegistry = (ServiceRegistry) testApplicationContext.getBean(ServiceRegistry.class);
+            nodeService = serviceRegistry.getNodeService();
+
+//            // Print out all available beans.
```

Comment:
You could wrap this with `logger.isTraceEnabled` and print the beans when trace is enabled

Reply:
like so:
         ```
   if (logger.isTraceEnabled()) {
                String[] beanDefinitionNames = testApplicationContext.getBeanDefinitionNames();
                for(String beanName : beanDefinitionNames){
                    logger.trace("{}", beanName);
                }
            }
```